SR2022/2022-04-02-competition-details-reminder.md

```diff
@@ -0,0 +1,46 @@
+
+Before then, we have a few important updates and reminders for you:
+
+## COVID Safety
+
+Life is beginning to get back to normal, and this means we are finally able to do in person events again! However, we must recognise that some people still have to be cautious, to protect their own health or their friends or family. As such, we ask that everyone follows [our COVID policy](https://studentrobotics.org/covid-19/) to help ensure we all keep safe, whilst still having lots of fun with robots!
+
+Please keep an eye on this as we get closer to the competition, as the content may change.
+
+## Southampton Marathon
+
+On Sunday 24th April, a marathon event is being held in Southampton. As a result there will be a number of road closures in the area, including on the university campus. Parking will also be limited. Keep an eye on the [competition page](https://studentrobotics.org/events/sr2022/competition/) on our website to see up-to-date information on how to reach the competition.
+
+Access to food sellers may also be restricted, so please plan accordingly.
+
+## Robot design considerations
+
+At the competition, all robots undergo a safety check. This not only confirms that the robot satisfies our safety regulations, but also the rules. Of note are a particular few we often see teams forgetting until the last minute:
+
+- For safety, please make sure the power button is obvious and easily accessible. In the event of an issue in the arena, we don't want to be hunting for the switch as we try and quickly turn off your robot.
+- Ensure your battery is protected, ideally housed in its own rigid container
+- Make sure you have a way to mount the flag. We will have a few spare mounts, but please do not rely on this.
+- Make sure your robot fits inside the size constraints. If it doesn't, it will have a meeting with the _saw of shame_.
+- Make sure there are 2 spare USB ports: 1 for your code, and 1 for our "Competition USB". This Competition USB tells your robot that it's in the competition and which corner it's starting in (and thus populates the `R.zone` property). You will not be able to compete without this.
+
+All of these and more can be found in our [safety regulations](https://studentrobotics.org/docs/kit/safety-regulations) and [rules](https://studentrobotics.org/rules).
+
+## Media consent
+
+As with previous years, we require you to sign a media consent form. These media consent forms allow us to use videos and photographs taken at the event for future marketing purposes. However, the process has changed slightly.
+
+[Download our Media Consent form](https://studentrobotics.org/resources/sr2022/media-consent.pdf)
+
+Please bring this form with you to the competition. We will have spares at reception, but this will allow you, your competitors and guests to proceed through faster. Depending on whether you give consent or not, we will issue you with a wristband or lanyard, so we ensure you are not in any of our media.
```

Comment:
```suggestion
Please bring this form with you to the competition. We will have spares at reception, but bringing a completed form will allow you, your competitors and guests to proceed through faster. Depending on whether you give consent or not, we will issue you with a wristband or lanyard. These will ensure those not providing consent are not in any of our media.
```

Reply:
This kind implies that 1 form might cover the whole team plus guests, which is inaccurate